eventlet/wsgi.py

```diff
@@ -456,6 +457,14 @@ def handle_one_request(self):
                 self.close_connection = 1
                 return
 
+            if transfer_encoding is not None:
+                self.wfile.write(
+                    b"HTTP/1.0 400 Bad Request - rfc9112 Content-Length and "
+                    b"Transfer-Encoding are not allowed together\r\n"
```

Comment:
That seems like a long status line -- any reason not to do something more like
``` python
msg = b"Content-Length and Transfer-Encoding are not allowed together\n"
self.wfile.write(
    b"HTTP/1.0 400 Bad Request\r\n"
    b"Connection: close\r\n"
    b"Content-Length: %d\r\n"
    b"\r\n%s" % (len(msg), msg))
```
? Maybe even throw in a `Content-Type`?

Reply:
Good idea, you are right, the body of the request is more suited to forward error details. Will update.

Reply:
I have a subsidiary observation, personally I prefer using HTTP/1.0 according to the [`max_http_version` option documentation](https://github.com/eventlet/eventlet/blob/master/eventlet/wsgi.py#L936C27-L936C101):

> Set to "HTTP/1.0" to make the server pretend it only supports HTTP 1.0. This can help with applications or clients that don't behave properly using HTTP 1.1.

However, during my development I seen that, in responses, the HTTP protocole is hardcoded and sometime the version is set to `HTTP 1.0` (the majority of the time) and sometime to `HTTP 1.1`:

- https://github.com/eventlet/eventlet/blob/master/eventlet/wsgi.py#L746
- https://github.com/eventlet/eventlet/blob/master/eventlet/wsgi.py#L28
- https://github.com/eventlet/eventlet/blob/master/eventlet/wsgi.py#L433
- https://github.com/eventlet/eventlet/blob/master/eventlet/wsgi.py#L439
- https://github.com/eventlet/eventlet/blob/master/eventlet/wsgi.py#L454
- and so on...

Still, according to the previous doc sentence, I think that everything should be set to HTTP/1.0, right?

This could be fixed later through a follow up patch.
Just want to confirm the current case related to my patch.